aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -443,6 +786,12 @@ def route(
         cors: Optional[bool] = None,
         compress: bool = False,
         cache_control: Optional[str] = None,
+        summary: Optional[str] = None,
+        description: Optional[str] = None,
+        responses: Optional[Dict[int, Dict[str, Any]]] = None,
+        response_description: Optional[str] = _DEFAULT_OPENAPI_RESPONSE_DESCRIPTION,
```

Comment:
I'm wondering here if we need to use `Optional` if we are adding a default value.
```suggestion
        response_description: str = _DEFAULT_OPENAPI_RESPONSE_DESCRIPTION,
```